packages/sqlduck/src/manager/database/duck-database-manager.test.ts

```
@@ -0,0 +1,87 @@
+import * as fs from 'node:fs';
+import path from 'node:path';
+
+import type { DuckDBConnection } from '@duckdb/node-api';
+import { sortBy } from 'es-toolkit';
+import { afterEach, beforeEach, describe } from 'vitest';
+
+import { createDuckdbTestMemoryDb } from '@/tests/utils/create-duckdb-test-memory-db.ts';
```

ESLint

[error] Unable to resolve path to module '@/tests/utils/create-duckdb-test-memory-db.ts'.

(import-x/no-unresolved)